src/devices/acurite.c

```diff
@@ -930,7 +930,7 @@ static int acurite_00275rm_callback(r_device *decoder, bitbuffer_t *bitbuf)
             else if (probe == 2) {
                 ptempc = (((signal[0][7] & 0x0f) << 8) | signal[0][8]) * 0.1 - 100;
                 data = data_append(data,
-                        "ptemperature_C",  "Celsius",      DATA_FORMAT,    "%.1f C",  DATA_DOUBLE, ptempc,
+                        _X("temperature_C", "ptemperature_C"),  "Celsius",      DATA_FORMAT,    "%.1f C",  DATA_DOUBLE, ptempc,
```

Comment:
This (and below) is actually a distinct value from the probe, not the base temp/hum value. It's a collision with the base value.

Reply:
Got it. Would `temperature_1_C` and `temperature_2_C` be more appropriate?

Reply:
not sure, so far I only normalized the keys that sorely stood out.

Reply:
I had a look at all of them and there were only a couple others that may benefit from clarification. I think the `temperature_<n>_C` approach is the way to go for this one since it seems to be a device that can report multiple temperatures from different probes.